Overview.bs

```diff
@@ -7,7 +7,6 @@ Group: htmlwg
 Repository: w3c/html-ruby
 TR: https://www.w3.org/TR/html-ruby-extensions/
 URL: https://w3c.github.io/html-ruby/
-Previous Version: https://www.w3.org/TR/2014/NOTE-html-ruby-extensions-20140204/
```

Comment:
Shall we use "none" for the `Level` metadata?  Otherwise the URL will not match with the one in `/TR`.

Also, I believe we need to update the templates for the SoTD of the HTML WG -> https://github.com/speced/bikeshed/tree/main/bikeshed/spec-data/readonly/boilerplate/html

Do you know which document I should modify in BikeShed?

Reply:
> Shall we use "none" for the `Level` metadata? Otherwise the URL will not match with the one in `/TR`.

Fixed in https://github.com/w3c/html-ruby/pull/3/commits/df3ea11061960908894986dd643e668943efc69e


> Also, I believe we need to update the templates for the SoTD of the HTML WG -> https://github.com/speced/bikeshed/tree/main/bikeshed/spec-data/readonly/boilerplate/html

These files are no longer active. Bikeshed now stores its templates in https://github.com/speced/bikeshed-boilerplate/

Either way, If you read the content of these templates, despite being in an "html" folder, they're about the old Web Platform WG. For the current HTMLWG, I've set up templates in the "htmlwg" folder:

https://github.com/speced/bikeshed-boilerplate/tree/main/boilerplate/htmlwg

I think these are appropriate for this spec, and the correct status gets automatically picked based on the `Status` metadata.

So I think we're good.

Reply:
This is very thoughtful. Thanks, Florian!